tests/accuracy_tests/sim_for_alphapeel_accu_test/sim_for_metafounder_accu.R

```diff
@@ -0,0 +1,267 @@
+rm(list = ls())
+# install.packages(pkg = "AlphaSimR")
+library(AlphaSimR)
+
+# ---- Two (or more) sub-populations A and B which cross in later generations ----
```

Comment:
@RosCraddock great reuse of the simulation code with two sub-populations;)

Reply:
We probably want to drop phenotypic and genetic values here to make it smaller.

Reply:
As in, no need to collect them

Reply:
Yes, sorry - I forgot to remove the redundant code!